specs/wp-post-editor-spec.js

```diff
@@ -132,8 +132,9 @@ test.describe( `[${host}] Editor: Posts (${screenSize})`, function() {
 
 							test.it( 'Can see the default publicise message', function() {
 								let postEditorSidebarComponent = new PostEditorSidebarComponent( driver );
+
 								postEditorSidebarComponent.publicizeMessageDisplayed().then( function( messageDisplayed ) {
-									assert.equal( messageDisplayed, '', 'The publicize message is not defaulting to empty' );
+									assert.equal( messageDisplayed, blogPostTitle, 'The publicize message is not defaulting to empty' );
```

Comment:
Since the publicize message is not empty anymore, change the error message so that we know we're expecting it to be the blog post title now.